database-portfolio/src/main/webapp/script.js

```diff
@@ -13,42 +13,77 @@
 // limitations under the License.
 //
 
+async function getUserLoginStatus() {
+  const url = '/login';
+  let response = await fetch(url);
+  let text = await response.text();
+  return text;
+}
+
 /** Loads and displays comments according to a limit */
-function loadCommentSection(commentLimit) {
-    // clear previous elements
-    var currentDOM = document.getElementsByClassName('indiv-comments');
-    if (currentDOM != null) {
-        console.log(currentDOM);
-        Array.from(currentDOM).forEach(function (element) {
+async function loadCommentSection(commentLimit) {
+    let userLoginInfo = await getUserLoginStatus();
+    let isLoggedIn = "Y" === userLoginInfo[0];
+
+    // clear previous elements to refresh login status
+    var loginDOM = document.getElementsByClassName('login');
+    if (loginDOM != null) {
+        Array.from(loginDOM).forEach(function (element) {
```

Comment:
Same comment as below. Use a different type of loop, and don't use anonymous functions.